perception/ground_segmentation/src/scan_ground_filter_nodelet.cpp

```diff
@@ -91,6 +150,202 @@ void ScanGroundFilterComponent::calcVirtualGroundOrigin(pcl::PointXYZ & point)
   point.z = 0;
 }
 
+void ScanGroundFilterComponent::gridScan_classifyPointCloud(
```

Comment:
I think lower camel case is better because other functions are used lower camel case.
Also, this function has deep nest. Can you remove deep nest by make sub-function or something?

Reply:
thanks for your comment. I fixed it.